### ISSUE TITLE
playsound command on Stage/Quest Completion

### ISSUE THREAD

**Issue by HamburgerJackson** [PikaMug/Quests#1574]
Hi,

I am wondering how I would create an action to execute the playsound command on a player, so that when a quest stage is completed, it would play a specific sound to notify the user?

I've been toying with the editor and attaching the action to the Finish Stage condition, but have not had luck. If this isn't possible, please take this as a suggestion.

Thanks very much.

**Comment by PikaMug** [comment]
In the Actions Editor, your options are:

- Effects -> Sound Effects - > Add sound effect
- Player -> Execute commands (runs from console but you can use `<player>` to represent the player)
- Denizen script

For more information, see https://github.com/PikaMug/Quests/wiki/Casual-%E2%80%90-Action-Editor

**Comment by HamburgerJackson** [comment]
Thank you! I struggled hard for some reason but it was actually pretty easy. For anyone else wondering, this is what I did.

Enter the actions editor (/quests actions), and go to 'create new action -> player -> execute commands' and entered:
**playsound minecraft:ui.toast.challenge_complete ambient <player>**

I then saved it, entered the quest editor (/quests editor) and navigated to 'edit quest -> edit stages -> stage 1 -> actions -> finish action'

In the list of actions, I selected the custom action that I created previously (I called it StageEnd). The wiki explains all of this pretty well. I just messed up a bit with it. Thank you for the clarification PikaMug.
